vale/Vocab/OSS-Docs/accept.txt

```diff
@@ -0,0 +1,68 @@
+Docusaurus
```

Comment:
Some of these are MM specific, but it's probably not too big of a deal. Also eventually Vale and it's GitHub action will support splitting out these Vocab directories. When that's supported we can have these kinds of things in a central repository that all of our projects can leverage.

Reply:
ooooh that would be so cool

Reply:
It looks like [the GitHub action was updated to `v1.5.0`](https://github.com/errata-ai/vale-action/releases/tag/v1.5.0) so we can go ahead and organize our Vocabs into various directories and share them across repositories. 🎉